pkg/agent/client.go

```diff
@@ -455,7 +455,7 @@ func (a *Client) remoteToProxy(connID int64, ctx *connContext) {
 
 	for {
 		n, err := ctx.conn.Read(buf[:])
-		klog.V(4).InfoS("received data from remote", "bytes", n, "connID", connID)
```

Comment:
Default log level of 4 was originally only supposed to be in the test client. Apparently I missed that in a recent review and we switched to 4 everywhere. Still trying to decide if its too late to switch back (backward compatibility problems). For now going to go ahead and approve. This switch still adheres to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md as we can (and should) consider this to be trace level logging.